### DISCUSSION TITLE
I don't think it's an ISSUE BUT

### DISCUSSION THREAD

**Discussion by Wjplatformer** [TomSchimansky/CustomTkinter#672] [General]
![image](https://user-images.githubusercontent.com/82447031/204291375-ee6e9901-4b07-4ca7-b450-5316bec27d20.png)
yeah
erm
![image](https://user-images.githubusercontent.com/82447031/204291419-b3487367-9499-4af1-b2d2-088fb56ef05c.png)

**Comment by TomSchimansky** [comment]
What's your intention with this post?

**Reply by Wjplatformer** [reply]
not really anything. I was setting relwdith and relheight to 100 for both for a customTKFrame widget, and it gave an error saying it executed correctly??? Mostly it was a joke as I put 100 on accident, but I was wondering why there would be such an issue popping up?
